test/tests/functional/pbs_power_provisioning_cray.py

```diff
@@ -830,16 +768,17 @@ def test_last_used_time_node_sort_key(self):
                 if i > 1:
                     self.server.manager(MGR_CMD_SET, NODE, {
                         'state': 'offline'}, id=n['id'])
-                i += 1
-        a = {'node_sort_key': '"last_used_time LOW " ALL'}
+                i = i + 1
+        a = {'node_sort_key': '"last_used_time LOW" ALL'}
         self.scheduler.set_sched_config(a)
         jid = self.submit_job(
             1, {'Resource_List.select': '1:ncpus=1',
                 'Resource_List.place': 'excl'})
         self.server.expect(JOB, {'job_state': 'F'}, id=jid, extend='x')
-        status = self.server.status(JOB, id=jid, extend='x')
+        status = self.server.status(JOB, 'exec_vnode', id=jid, extend='x')
         exec_vnode = status[0]['exec_vnode']
         node1 = exec_vnode.split(':')[0][1:]
+        print node1
```

Comment:
This looks like a debugging statement.  Either remove it or turn it into a self.logger.info() and add more explanation to it.